docs/provisioning/external-silos.md

```diff
@@ -0,0 +1,154 @@
+      ```
+3. Connect an existing k8s cluster to Azure Arc by creating an Azure Arc-enabled Kubernetes resource named `<Azure-Arc-enabled-k8s-resource-name>`. **This step should be performed by the Silo Admin, since it requires access to the k8s cluster (via the kube config file). It also requires Contributor role (at least) in the resource group `<connected-cluster-resource-group>` created in the previous step.** 
+    - Enter the following command.
+      ```bash
+      az connectedk8s connect --name <Azure-Arc-enabled-k8s-resource-name> --resource-group <connected-cluster-resource-group>
+      ```
+    - If the default kube config and context do not point to the k8s cluster, then the `--kube-config` and `--kube-context` parameters can be used to specify the correct values.
+
+
+### C. Deploy the Azure ML extension on the k8s cluster
+Detailed instructions for this phase, including current limitations, steps for verification or for slightly different use cases can be found [there](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-deploy-kubernetes-extension?tabs=deploy-extension-with-cli). Here is a summary, which should be all you need.
+
+**This step should be run by the FL Admin.** The Azure CLI extension `k8s-extension` with version >= 1.2.3 is required. It can be installed _via_ `az extension add --name k8s-extension`. 
+
+To deploy the Azure ML extension on the k8s cluster, enter the following command.
+```bash
+az k8s-extension create --name <extension-name> --extension-type Microsoft.AzureML.Kubernetes --config enableTraining=True --cluster-type connectedClusters --cluster-name <Azure-Arc-enabled-k8s-resource-name> --resource-group <connected-cluster-resource-group> --scope cluster
+```
+where `<extension-name>` can be chosen arbitrarily.
+
+The deployment can be verified by the following.
+```bash
+az k8s-extension show --name <extension-name> --cluster-type connectedClusters --cluster-name <Azure-Arc-enabled-k8s-resource-name> --resource-group <connected-cluster-resource-group>
+```
+
+In the response, look for `"name"` and `"provisioningState": "Succeeded"`. Note that this step can take 10-15 minutes and will show `"provisioningState": "Pending"` at first.
+
+
+### D. Attach the Arc cluster to the orchestrator workspace
+
+**This step should be run by the FL Admin.** The `ml` Azure CLI extension (_a.k.a._ Azure ML CLI v2) will be required. It can be installed _via_ `az extension add --name ml`. See [over there](https://learn.microsoft.com/en-us/azure/machine-learning/how-to-configure-cli?tabs=public) for more details on installation.
+
+1. First, we'll need to create a user-assigned identity (uai) for the soon-to-be-created Azure ML attached compute. We recommend creating it in the workspace resource group, and using a name of the form: `uai-<azureml-compute-name>`, where `<azureml-compute-name>` will be reused shortly and will be the name of the attached compute in Azure ML.
```

Comment:
I would suggest simplifying the wording. Remove "First" (because we're on point 1), and just provide the command with a minimal instruction set.

Example:

1. Create a user-assigned identity (UAI) that will be assigned to the Azure ML attached compute:

    ````bash
    put the command here
    ````

Reply:
done